api/firmware/messages/bitboxbase.proto

```diff
@@ -27,6 +27,11 @@ message BitBoxBaseHeartbeatRequest {
     INITIAL_BLOCK_SYNC = 1;
     DOWNLOAD_UPDATE = 2;
     OUT_OF_DISK_SPACE = 3;
+    REDIS_ERROR = 4;
```

Comment:
In the `func (device *Device) BitBoxBaseHeartbeat()` docstring, please enumerate those too and specify that they are available with "firmware version >= 4.4.0". Since we don't have a comprehensive api documentation in one place yet, either we need the checks in the code (`if !device.version.AtLeast(...) { return error... }`), or document it in the docstring.